packages/docs/pages/select.tsx

```diff
@@ -1,399 +1,499 @@
-                  options={[
-                    { value: 1, content: 'Option' },
-                    { value: 2, content: 'Option' },
-                    { value: 3, content: 'Option' },
-                    { value: 4, content: 'Option' },
-                    { value: 5, content: 'Option' },
-                    { value: 6, content: 'Option' },
-                    { value: 7, content: 'Option' },
-                    { value: 8, content: 'Option' },
-                    { value: 9, content: 'Option' },
-                    { value: 10, content: 'Option' },
-                    { value: 11, content: 'Option' },
-                    { value: 12, content: 'Option' },
-                  ]}
-                  placeholder="Larger"
-                  required
-                />
-              </Grid>
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-          <Panel header="Disabled">
-            <Text>
-              It is possible to disable the entire select component, similar to how you would disable a native HTML
-              select element, by using the <Code primary>disabled</Code> property.
-            </Text>
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          <Code primary>Select</Code> give merchants the ability to make a single selection or multiple selections from
+          a list of options.
+        </Text>
 
-            <CodePreview>
-              {/* jsx-to-string:start */}
-              <Form>
-                <FormGroup>
-                  <Select
-                    disabled
-                    label="Select"
-                    maxHeight={350}
-                    onOptionChange={() => null}
-                    options={[
-                      { value: 1, content: 'Option' },
-                      { value: 2, content: 'Option' },
-                      { value: 3, content: 'Option' },
-                      { value: 4, content: 'Option' },
-                    ]}
-                    placeholder="Larger"
-                    required
-                  />
-                </FormGroup>
-              </Form>
+        <Text bold>When to use:</Text>
+        <List>
+          <List.Item>When a user needs to make selections in a form.</List.Item>
+        </List>
+      </Panel>
 
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-          <Panel header="Action">
-            <Text>
-              Accepts an <Code>action</Code> object to display at the end of the list.
-            </Text>
+      <Panel header="Implementation" headerId="implementation">
+        <ContentRoutingTabs
+          id="implementation"
+          routes={[
+            {
+              id: 'basic',
+              title: 'Basic',
+              render: () => (
+                <Fragment key="basic">
+                  <Text>
+                    <Code primary>Select</Code> are typeable inputs with selectable options.
```

Comment:
I updated `selectable dropdown items` to `selectable options` is that ok? suggestions?

Reply:
Good to me.